src/live.asp.net/AppSettings.cs

```diff
@@ -16,5 +16,7 @@ public class AppSettings
         public string AzureStorageBlobName { get; set; }
 
         public string AzureStorageContainerName { get; set; }
+
+		public string CDNUrl { get; set; }
```

Comment:
If you look closely here in the last line you will recognize there's one space missing before `public`, perhaps the Tab Size in VS is different